src/components/newInvoice/NewInvoiceForm.js

```diff
@@ -21,6 +22,16 @@ const NewInvoiceForm = () => {
     // list of possibles contractors, data get from db.json file
     const [contractors, setContractors] = useState([])
 
+    // contractors list updated
+    const [updateContr, setUpdateContr] = useState(false)
```

Comment:
this var name is miss leading. I would suggest `[isContractorFormOpen, toggleContractorFormOpen]` or `setIsContractorFormOpen`